src/server/routes/api/document/index.js

```diff
@@ -118,9 +127,16 @@ http.get('/', function( req, res, next ){
     tryPromise( () => checkApiKey(apiKey) )
     .then( () => loadTable('document') )
     .then( t => {
-      let { table, conn } = t;
-
-      return table.skip(offset).limit(limit).run(conn);
+      let { table, conn, rethink: r } = t;
+      return table
+        .skip(offset)
+        .limit(limit)
+        .filter(r.row('secret').ne(DEMO_SECRET))
+        .pluck(['id'])
```

Comment:
This will break the admin panel, as it previously expected the array of doc JSON. Was this on purpose?

Reply:
OK this is fine. I'll update the admin and these fields in another PR.